python/tchannel/thrift/transport/tornado_base.py

```diff
@@ -74,18 +77,20 @@ def recv_call(self):
             "recv_call() not supported for Tornado. Use readFrame()."
         )
 
+    @tornado.gen.coroutine
     def _flush_internal(self, endpoint, response, seqid):
         buff = TTransport.TMemoryBuffer()
 
         # This is so dirty, /I can't even.../
         binary = TBinaryProtocol.TBinaryProtocol(buff)
-        if response.message_type == Types.CALL_RES:
+        if isinstance(response, Response):
```

Comment:
Wait, what happens when it's not a response object?

Do we have it returning Response and Message at the same time?